src/xUnitToJUnit/xUnitToJUnit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
Can you also remove the [GitHub Actions step installing .NET 5](https://github.com/gabrielweyer/xunit-to-junit/blob/b925cbf321f3f4accb2051e9677cee92354aac1a/.github/workflows/build.yml#L42-L45)? This won't be required any more.